templates/about.html

```diff
@@ -3,7 +3,7 @@
 
 {% block title %}About HTTP Archive{% endblock %}
```

Comment:
```suggestion
{% block title %}About the HTTP Archive{% endblock %}
```

Should we review all the titles?